src/tests/HomePage.test.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import renderer from 'react-test-renderer';
+import '@testing-library/jest-dom';
+import { render, screen, waitFor } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import axios from 'axios';
+import store from '../redux/configureStore';
+import HomePage from '../Components/HomePage';
+// import { dogActions, getDogs } from '../Redux/mainPage';
+
```

Comment:
- [ ] Kindly remove all unused or unnecessary codes from your file. It helps keep the codebase clean. 👍